currency-converter/src/components/Form.tsx

```diff
@@ -1,5 +1,116 @@
+import { IoIosSwap } from 'react-icons/io';
+import Select, { StylesConfig } from 'react-select';
+
 export default function Form() {
+  const currencies = [
+    { value: 'eur', label: 'EUR' },
+    { value: 'usd', label: 'USD' },
+    { value: 'gbp', label: 'GBP' },
+    { value: 'jpy', label: 'JPY' },
+  ];
+
+  const customStyles: StylesConfig = {
```

Comment:
move these styles and dropdown to a separate component, and use it generically

Reply:
Detals: A new component will be made with name "Dropdown" and it will take 2 things in props. 
1. List of the data to be shown such as, 
const currencies = [
    { value: 'eur', label: 'EUR' },
    { value: 'usd', label: 'USD' },
    { value: 'gbp', label: 'GBP' },
    { value: 'jpy', label: 'JPY' },
  ];
  2. onChange function, this will be triggered whenever user changes the currency. (This can be skipped for now as you're just making UI rn)

Reply:
And styles for the component and all other extra things should be done in that new dedicated component, Our parent component should have just one line like this,
<Dropdown  data={currencies}/> and it should print dropdown.

Reply:
ok for onchange has been implemented in third branch but that need some changes after UI is completed.